cypress/integration/frontsite/homepage.test.spec.js

```diff
@@ -18,7 +18,7 @@ describe('frontsite homepage', () => {
       .and('include', 'sort=created')
       .and('include', 'order=desc')
       .and('include', 'doctype%3ABOOK')
-      .and('include', 'medium%3AELECTRONIC_VERSION');
+      .and('include', 'medium%3AE-BOOK');
```

Comment:
did you check if this works everywhere with the hyphen? Aggregations and searches mainly?

Reply:
yes it works